README.md

```diff
@@ -1,30 +1,5 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
```

Comment:
You did a great job. That's fine. 👍  I noticed that you did not remove the readme file's comment lines. Please remove it. It will look your file cool.